### ISSUE TITLE
Bookmarks Manager corrupted

### ISSUE THREAD

**Issue by PeterinScotland** [operasoftware/bookmarks-manager-extension#10]
Help!  My Bookmarks Manager has disappeared because it's supposedly corrupted.  It's not available on the Opera site any more because they evidently have their own Bookmarks Manager now.  I downloaded the zip file from Github but can't work out how to install it to Opera or how to find the bookmarks I had yesterday.  I like the pull down list, and I had a long list of bookmarks that are not in the built-in Opera list.  I'm not even sure how I got all those bookmarks in the Opera version.

**Comment by brucelawson** [comment]
Peter, if you open the page View > Extensions in Opera, you should just be able to drag the zip file onto the page and drop i there to install it.

**Comment by PeterinScotland** [comment]
Bruce

Thank you.  It didn't work with the zip file (due to nested folders 
inside), but I was able to drag the extracted folder containing the 
files into that page and it appears it may have installed.

Now, please, I need to know where my bookmarks file is from before your 
extension became corrupted - the newly installed extension has not yet 
found the bookmarks

Kind regards

Peter Reynolds
=========

**Comment by PeterinScotland** [comment]
Do you know where I can find the old bookmarks file now I have reinstalled the extension?

**Comment by brucelawson** [comment]
I don't think you can get them now, because they were stored in the extension which was corrupted. Do you want me to find out how to import them from old Opera (I assume O12) into the native bookmarking facility in the latest Opera?

Meanwhile, I'll retire this extension.

**Comment by NeoRCM** [comment]
Yes please, Bruce. I made an account just to ask you this. I used your extension for a long time and had many important bookmarks there that I didn't have anywhere else. I would really appreciate it if you could tell me how to recover them. My Opera version is 34.0.2036.16 if it matters.

**Comment by PeterinScotland** [comment]
In my case I never had Opera prior to Chromium (Opera 15?).  I was attracted by the publicity and the idea of having something like Chrome without being spied on by Google.  I still use it though I use some other browsers too (the lack of Print Preview/settings was one thing that stopped me using Opera for everything).  Very recently Chrome has started crashing banking sessions and the like.  I don't know whether it's an Opera problem or some kind of extension issue.

**Comment by PeterinScotland** [comment]
PS Banking issue disappeared when I disabled an Images On/Off extension that I've had for a long time.  No idea why it suddenly started, nor why that extension should have been a problem as I wasn't using it during banking.

**Comment by brucelawson** [comment]
The bookmarks were saved inside the extension, so if it's corrupted,
they're lost. I'm sorry.

For anyone else who wants to import extensions into Opera 20+ from Opera
12, paste this into your address bar browser://settings/?search=import
